### PR TITLE
Add expanded/compact widget toggle to app-header

### PR DESCRIPTION
**Pull request by thevoiceofzeke**
### Screenshots
screen shot 2016-08-09 at 12 01 24 pm
screen shot 2016-08-09 at 12 01 31 pm
screen shot 2016-08-09 at 2 04 43 pm

### CODE REVIEW

File: angularjs-portal-home/src/main/webapp/my-app/layout/partials/home-header.html [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
   <div ng-controller="ToggleController">
-    <md-menu-item ng-if='modeIs(MODES.EXPANDED)'>
-      <md-button class="md-accent" ng-click="switchMode(MODES.COMPACT)">Show Compact Mode</md-button>
-    </md-menu-item>
-    <md-menu-item ng-if='modeIs(MODES.COMPACT)'>
-      <md-button class="md-accent" ng-click="switchMode(MODES.EXPANDED)">Show Expanded Mode</md-button>
-    </md-menu-item>
+    <md-switch class="md-accent md-hue-2 app-header-toggle" ng-model="expandedMode" aria-label="Toggle expanded widget layout" ng-change="toggleMode(expandedMode)">
+      <span hide-sm hide-xs>Show expanded mode</span>
+      <span hide-gt-sm>Show Expanded</span>
```
Comment (apetro) [inline angularjs-portal-home/src/main/webapp/my-app/layout/partials/home-header.html]: "Expand widgets", maybe?

**Comment by timlevett** [comment]
Anyway to get rid of the scrolling things in the dropdown?

**Comment by thevoiceofzeke** [comment]
@timlevett It was taken care of in a merge to frame:

https://github.com/UW-Madison-DoIT/uw-frame/pull/266

**Comment by timlevett** [comment]
Gotcha, then 👍 from me

**Comment by vertein** [comment]
👍

**Pull request merged**